modules/files/files.go

```diff
@@ -173,6 +173,17 @@ func PathContainsHiddenFileOrFolder(path string) bool {
 	return false
 }
 
+// PathIsTerraformVersionFile returns true if the given path is the special '.terraform-version' file used by the [tfenv](https://github.com/tfutils/tfenv) tool.
+func PathIsTerraformVersionFile(path string) bool {
+	pathParts := strings.Split(path, string(filepath.Separator))
+	for _, pathPart := range pathParts {
+		if strings.Contains(pathPart, ".terraform-version") {
+			return true
+		}
+	}
```

Comment:
NIT: If `.terraform-version` is a file, wouldn't it be the final part of the path? And if so, could this be simplified to `filepath.Base(path) == ".terraform-version"`?

Reply:
Fixed!